docs/source/pages/documentation.md

```diff
@@ -78,16 +78,16 @@ The optional arguments `overwrite_old_files`, `transfer_verbosity` and `show_tra
 
 ### Example
 
-An example call to `make-config-file` below creates a new project called `my_first_project`, sets the *local* project path to `/path/to/my/project`, the *central* path (to a remote Linux server) to `/nfs/nhome/live/username/`, the required SSH configurations, and indicates that *behavioural*, _electrophysiological_ and *histological* data will be used on this machine for this project.
+An example call to `make-config-file` below creates a new project called `my_first_project`, sets the *local* project path to `/path/to/my/project/my_first_project`, the *central* path (to a remote Linux server) to `/nfs/nhome/live/username/my_first_project`, the required SSH configurations, and indicates that *behavioural*, _electrophysiological_ and *histological* data will be used on this machine for this project.
```

Comment:
The example path `/path/to/my/project/my_first_project` is a bit confusing, because the "project" appears twice. I think it would be better to have `/path/to/my_first_project`, or even `/path/to/my_projects/my_first_project`. The latter also better matches the Windows example path given in the "Initial setup" section.

If you change it, don't forget to also update the other occurrences of this path below.